httpserver/test_helpers.go

```diff
@@ -0,0 +1,19 @@
+// Package httpserver provides HTTP API handlers for HTCondor operations.
+package httpserver
+
+import (
+	"crypto/rand"
+	"fmt"
+
+	"github.com/bbockelm/cedar/security"
+)
+
+// GenerateSigningKey generates a new signing key for token generation
+// Returns the key content as bytes
+func GenerateSigningKey() ([]byte, error) {
```

Comment:
Inspect the `security` package to see if it has any key generation utilities.  If so, use those and remove this one.